docs/cosmology/dev.rst

```diff
@@ -147,7 +147,7 @@ parameter and change any constructor argument. For example, see
     class FlatFLRWMixin(FlatCosmologyMixin):
         ...
 
-        Ode0 = FLRW.Ode0.clone(derived=True)  # now a derived param.
+        Ode0 = FLRW.parameters["Ode0"].clone(derived=True)
```

Comment:
So, this is a pretty big API break too...

Reply:
No change made as of yet, just a change of focus. `FLRW.Ode0` still works.

Reply:
Resolving given https://github.com/astropy/astropy/pull/15168#issuecomment-1762378211.